.gitignore

```diff
@@ -123,6 +123,7 @@ owtf/webapp/src/Report/templates/*
 # Don't check auto-generated stuff into git
 owtf/webapp/coverage
 owtf/webapp/stats.json
+*webui/
```

Comment:
I don't think this is needed since this was created when the install script fetches the report templates. This was fixed in a commit yesterday.